package.json

```diff
@@ -7,11 +7,20 @@
     "example": "example"
   },
   "dependencies": {
+    "@mapbox/node-pre-gyp": "^1.0.10",
     "nan": "2.14.*",
     "typescript": "^3.8.3"
   },
   "scripts": {
-    "test": "mocha test/*"
+    "test": "mocha test/*",
+    "install": "node-pre-gyp install --fallback-to-build"
+  },
+  "binary": {
+    "module_name": "node-yara",
+    "module_path": "./lib/binding/",
+    "host": "https://github.com/",
+    "remote_path": "/Automattic/node-yara/raw/master/binaries/{platform}/",
+    "package_name": "yara.tar.gz"
```

Comment:
This will fetch `tar.gz` archives from https://github.com/Automattic/node-yara/tree/master/binaries, example:

```
$ npm i

> @automattic/yara@2.4.0 install
> node-pre-gyp install --fallback-to-build

node-pre-gyp info it worked if it ends with ok
node-pre-gyp info using node-pre-gyp@1.0.10
node-pre-gyp info using node@16.16.0 | darwin | x64
node-pre-gyp info check checked for "/Users/macbre/a8c/repos/node-yara/lib/binding/node-yara.node" (not found)
node-pre-gyp http GET https://github.com/Automattic/node-yara/raw/master/binaries/darwin/yara.tar.gz
node-pre-gyp info install unpacking yara.node
node-pre-gyp info extracted file count: 1 
[@automattic/yara] Success: "/Users/macbre/a8c/repos/node-yara/lib/binding/node-yara.node" is installed via remote
node-pre-gyp info ok 

up to date, audited 121 packages in 1s

22 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
```